helm/defectdojo/templates/django-deployment.yaml

```diff
@@ -22,6 +22,10 @@ spec:
         defectdojo.org/component: django
         app.kubernetes.io/name: {{ include "defectdojo.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
+      annotations:
+      {{- with .Values.django.annotations }}
```

Comment:
Could you update the values.yml with empty annotations for all three changes  ?  It's usually done as a helper so that looking at the values.yml only ppl know what they can configure.